includes/class-newspack.php

```diff
@@ -153,6 +153,8 @@ private function includes() {
 
 		// Filter by authors in the Posts page.
 		include_once NEWSPACK_ABSPATH . 'includes/author-filter/class-author-filter.php';
+
+		\Newspack\CLI\Initializer::init();
```

Comment:
When running the CLI command, I get `PHP Fatal error:  Uncaught Error: Class "Newspack\CLI\Initializer" not found` error. The `includes/cli/class-initializer.php` file is not required anywhere.